Makefile

```diff
@@ -1,6 +1,6 @@
-KUBE_VERSION ?= 1.23
+KUBE_VERSION ?= 1.24
 KIND_CONFIG ?= ./testdata/kind-$(KUBE_VERSION).yaml
-CERT_MANAGER_VERSION ?= 1.6.1
```

Comment:
We do have renovate configuration that should take care of this. I wonder why it didn't work? :thinking:

Reply:
Looks like this will now be fixed in #548